leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -370,25 +389,31 @@ var ElementInspector = Backbone.View.extend({
      * This function is called on change for #init-sat-value,
      */
     initSatValueChanged: function(event) {
```

Comment:
You don't need the event if you don't use it. Its one of the "unspoken" parameters. You don't need to explicitly have it in the params unless you want to access it. (Kind of like "options" in the initialize function if that is a helpful way to think about it")

Reply:
I think that we might need it because `initSatValueChanged()` calls `updateHTML()` which calls `validityCheck()` which actually requires event...

Reply:
This is actually maybe a much bigger issue that I should have caught way sooner but none of the class methods should actually touch the html...All html handling stuff should be done in the view, and then the model can contain model logic

Reply:
Basically anytime you need values from the html you should be getting those in the view and then passing them into a function in the model if you need to check validity/logic/etc

Reply:
This might be a stretch but thoughts on putting multiple function calls in your events? I know @YesugenB figured out how to do it...I am noticing that updateHTML() is usually called at the end of functions, and validityCheck() is then called in the first line of updateHTML(). Basically having a chain that goes "initSatValueChanged(), validityCheck(), updateHTML()" that way you don't have to pass the event parameter through 3 functions

Reply:
I changed events to this 
<img width="425" alt="Screen Shot 2021-06-30 at 12 32 25 PM" src="https://user-images.githubusercontent.com/59470488/123998181-3c528300-d99f-11eb-9379-d40be8c913cf.png">
and removed the event param from the initSatValueChanged and updateHTML (and commented out the function call for `updateHTML()` at initSatValueChanged and `validityCheck()` for updateHTML). 

Is this what you meant? I'm a little confused.

Reply:
Yes - for the most part. I guess I was thinking if the first two functions only use the event param to pass it on to the last one, you could put all 3 in events and pass it directly to only the last function if that makes sense

Reply:
There are a few other functions that call updateHTML() and need to call validityCheck(). Should we do the same for those 2 functions?